### PR TITLE
Generated the wordlist from more texts.

### PR DESCRIPTION
**Pull request by rmlockwood**

### CODE REVIEW

**Comment by keyman-server** [comment]
This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR.

File: release/sil/sil.glk-arab.ptwl1/sil.glk-arab.ptwl1.kpj [hunk 1]
```diff
@@ -19,12 +19,12 @@
       <ID>id_f008039edc7c2114f0c7b27dfdb93884</ID>
       <Filename>sil.glk-arab.ptwl1.model.kps</Filename>
       <Filepath>source\sil.glk-arab.ptwl1.model.kps</Filepath>
-      <FileVersion>1.0.1</FileVersion>
+      <FileVersion>1.0.2</FileVersion>
```
Comment (darcywong00) [inline release/sil/sil.glk-arab.ptwl1/sil.glk-arab.ptwl1.kpj:22]: This file looks like it belongs on a separate PR

**Comment by rmlockwood** [comment]
Sorry. Could you remove it from the pull request or tell me how I do it?

 

Thanks,

Ron

**Comment by rmlockwood** [comment]
Done.

**Comment by rmlockwood** [comment]
Actually, now it’s done.

File: release/sil/sil.glk-arab.ptwl1/sil.glk-arab.ptwl1.kpj (hunk 1 above)
Comment (darcywong00) [reply release/sil/sil.glk-arab.ptwl1/sil.glk-arab.ptwl1.kpj:22]: One way to undo this change:
1. check out your `sil.bcc-arab` branch
2. edit this file back to 1.0.1
3. commit and push the branch.

**Comment by rmlockwood** [comment]
done

**Comment by darcywong00** [comment]
> done

Can you also revert line 27. in that file back to `1.0.1`?

**Comment by rmlockwood** [comment]
done
